learning_paths/models.py

```diff
@@ -185,3 +196,43 @@ def estimated_end_date(self):
         if self.learning_path.duration_in_days is None:
             return None
         return self.created + timedelta(days=self.learning_path.duration_in_days)
+
+
+class LearningPathGradingCriteria(models.Model):
+    """
+    Grading criteria for a learning path.
+
+    .. no_pii:
+    """
+
+    learning_path = models.OneToOneField(
+        LearningPath, related_name="grading_criteria", on_delete=models.CASCADE
+    )
+    completion_threshold = models.FloatField(
+        default=100.0,
+        help_text="Completion percentage threshold for this learning path.",
+    )
+    expected_grade = models.FloatField(
+        default=75.0,
```

Comment:
Should we use the 0.0 - 1.0 scale for the completion and grade to make it consistent with the `weight` of each step?

Reply:
Makes sense. Updated it.